snippets/multisign.py

```diff
@@ -57,10 +58,10 @@
 if multisigned_tx_response.result["validated"]:
     print("The multisigned transaction was accepted by the ledger:")
     print(multisigned_tx_response)
-    if multisigned_tx_response.result["Signers"]:
+    if multisigned_tx_response.result["tx_json"]["Signers"]:
```

Comment:
nit: this would be easier to read with a helper variable for `multisigned_tx_response.result["tx_json"]["Signers"]`

Reply:
fixed in 4179077